multiqc/modules/sexdeterrmine/sexdeterrmine.py

```diff
@@ -128,6 +130,33 @@ def read_count_barplot(self):
             plot = bargraph.plot(self.sexdet_data, cats, config)
         )
 
+    def snp_rate_scatterplot(self):
+        """ Make a scatter plot showing relative coverage on X and Y chr
+        """
+        data = OrderedDict()
+        for sample in self.sexdet_data:
+            try:
+                data[sample] = {'x': self.sexdet_data[sample]['RateX'], 'y': self.sexdet_data[sample]['RateY']}
+            except KeyError:
+                pass
+
+        config = {
+            'id': 'sexdeterrmine-rate-plot',
+            'title': 'SexDetErrmine: Relative coverage',
+            'ylab': 'Relative Cov. on Y',
+            'xlab': 'Relative Cov. on X',
+            'xmax': 1.8,
+            'ymax': 1.2
+        }
+
+        self.add_section(
+            name = 'Relative Coverage',
+            anchor = 'sexdeterrmine-rates',
+            description = 'The coverage on the X vs Y chromosome, relative to coverage on the Autosomes.',
```

Comment:
It would be nice to have some `helptext` with a bit more explanation of what to look out for in this plot, if possible.

Reply:
Nice! I literally meant `helptext` though 😄

See https://multiqc.info/docs/#step-5-create-report-sections

Reply:
Updated, now looks like this (after clicking "Help"):

![image](https://user-images.githubusercontent.com/465550/82427092-eb2c4c00-9a88-11ea-8fde-7f7f500e4323.png)